app/src/main/java/org/fossasia/openevent/app/event/detail/EventDetailsActivity.java

```diff
@@ -50,20 +54,24 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_event_details);
 
-        ButterKnife.bind(this);
+        Event initialEvent = getIntent().getParcelableExtra(EVENT_KEY);
+
+        if(initialEvent == null) {
```

Comment:
Tell me, how can you assume that the event object was passed to you (even if it was) ? You have to check if it was actually past. If you are telling me to remove null check because we are definitely passing the event currently, remember that this is open source project. If there comes a point where someone forgets to pass a parameter to event detail activity, would you want the app to crash?

Always check nullable values - it's this strategy, not anything else